core/control-plane/control-plane-aggregate-services/src/main/java/org/eclipse/edc/connector/controlplane/services/policydefinition/PolicyDefinitionQueryValidatorFactory.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 Zichuan
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * SPDX-License-Identifier: Apache-2.0
+ * 
+ * Contributors:
+ *       Zichuan - initial API and implementation
```

Comment:
Please use the correct legal entity definition

Reply:
> Please use the correct legal entity definition

I work for a lab of a University. Since the lab is not a legal entity and that I cannot represent the University, I would leave my name for now. I would confirm with the Prof. about this during workday.

Reply:
you can also use "Contributors to the Eclipse Foundation" (ref. https://www.eclipse.org/projects/handbook/#ip-copyright-headers)

Reply:
You cannot use your name. You must use your employer's legal name.